nucleus/distributions/pom.xml

```diff
@@ -168,7 +168,12 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
Doesn't this make specifying the profile activation on each of payara, payara-web, payara-ml,and payara-web-ml redundant?
You've set it to uncategorically skip deployment here, and then also told it to skip deployment when the `deploy-internal` profile is activated.

Reply:
This is nucleus/distributions, not appserver, so only affects minimal

Reply:
🤦‍♂️
Well the point kind of stands - could we not just do a similar thing over in appserver/distributions rather than specifying it on each distribution there?